modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc

```diff
@@ -36,17 +36,20 @@ metadata:
   annotations:
     che.eclipse.org/oauth-scm-server: github
     che.eclipse.org/scm-server-endpoint: __<github_server_url>__ <2>
-    che.eclipse.org/scm-github-disable-subdomain-isolation: true <3>
 type: Opaque
 stringData:
-  id: __<GitHub_OAuth_Client_ID>__ <4>
-  secret: __<GitHub_OAuth_Client_Secret>__ <5>
+  id: __<GitHub_OAuth_Client_ID>__ <3>
+  secret: __<GitHub_OAuth_Client_Secret>__ <4>
 ----
 <1> The {prod-short} namespace. The default is `{prod-namespace}`.
 <2> This depends on the GitHub product your organization is using: When hosting repositories on GitHub.com or GitHub Enterprise Cloud, omit this line or enter the default `https://github.com`. When hosting repositories on GitHub Enterprise Server, enter the GitHub Enterprise Server URL.
-<3> This line is only added for GitHub Enterprise Server with disabled link:https://docs.github.com/en/enterprise-server/admin/configuration/hardening-security-for-your-enterprise/enabling-subdomain-isolation#about-subdomain-isolation[subdomain isolation] option. If the subdomain isolation option is enabled on GitHub Enterprise Server, you must either omit this annotation or set it to `false`.
-<4> The *GitHub OAuth Client ID*.
-<5> The *GitHub OAuth Client Secret*.
+<3> The *GitHub OAuth Client ID*.
+<4> The *GitHub OAuth Client Secret*.
++
+[IMPORTANT]
+====
+If you are using a GitHub Enterprise Server with disabled link:https://docs.github.com/en/enterprise-server/admin/configuration/hardening-security-for-your-enterprise/enabling-subdomain-isolation#about-subdomain-isolation[subdomain isolation] option, you must add `che.eclipse.org/scm-github-disable-subdomain-isolation` annotation set to `"true"`.
+====
```

Comment:
@vinokurig this is a nice note, but I would probably leave line 39, but set it to false by default

Reply:
done